src/components/text-input-with-copy-button/TextInputWithCopyButton.js

```diff
@@ -216,6 +218,7 @@ class TextInputWithCopyButton extends React.PureComponent<Props, State> {
         return (
             <div className={wrapperClasses} {...copyEvent}>
                 <TextInput {...inputProps} onFocus={this.handleFocus} />
+                {additionalButtons}
```

Comment:
Do we need a default value of null? What does `{undefined}` do here?

Reply:
`undefined` will not render anything

Reply:
Good to know thanks!